io.github.sameboy.SameBoy.metainfo.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <id>io.github.sameboy.SameBoy</id>
+  <name>SameBoy</name>
+  <summary>Game Boy and Game Boy Color emulator</summary>
+  <developer_name>Lior Halphon</developer_name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <description>
+    <p>SameBoy is an open source Game Boy (DMG) and Game Boy Color (CGB) emulator,
+written in portable C. It has an SDL frontend and a libretro core.
+It also includes a text-based debugger with an expression evaluator.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://dashboard.snapcraft.io/site_media/appmedia/2020/03/Bildschirmfoto_von_2020-03-16_01-01-41.png</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://dashboard.snapcraft.io/site_media/appmedia/2020/03/Bildschirmfoto_von_2020-03-16_00-58-39_sAm4YZR.png</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://dashboard.snapcraft.io/site_media/appmedia/2020/03/Bildschirmfoto_von_2020-03-16_01-00-07.png</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://dashboard.snapcraft.io/site_media/appmedia/2020/03/Bildschirmfoto_von_2020-03-16_01-02-12.png</image>
+    </screenshot>
+  </screenshots>
+  <releases>
```

Comment:
There's no point in including full release history given these versions were never available on Flathub in the first place.

Reply:
This is automatically generated and submitted upstream.